pkg/kfapp/ossm/ossm_manifests.go

```diff
@@ -1,44 +1,52 @@
 package ossm
 
 import (
+	"embed"
 	"fmt"
-	kftypesv3 "github.com/opendatahub-io/opendatahub-operator/apis/apps"
 	configtypes "github.com/opendatahub-io/opendatahub-operator/apis/config"
 	"github.com/opendatahub-io/opendatahub-operator/pkg/kfconfig/ossmplugin"
 	"github.com/opendatahub-io/opendatahub-operator/pkg/secret"
 	"github.com/pkg/errors"
+	"io/fs"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/rest"
-	"os"
 	"path"
 	"path/filepath"
 	"strings"
 )
 
+//go:embed templates
+var embeddedFiles embed.FS
+
 type applier func(config *rest.Config, filename string, elems ...configtypes.NameValue) error
 
 func (o *OssmInstaller) applyManifests() error {
 	var apply applier
 
 	for _, m := range o.manifests {
+		targetPath, err := m.targetPath(o.Name, o.Namespace)
+		if err != nil {
+			log.Error(err, "Error generating target path")
+		}
```

Comment:
Shouldn't we return error here? Otherwise we would continue with broken path and not really being able to apply anything.

Reply:
[1ce2892](https://github.com/maistra/opendatahub-operator/pull/5/commits/1ce2892d25dc3fa0bf60fe4b742672227c458560) added here